backend/de.metas.business.rest-api-impl/src/test/resources/de/metas/rest_api/v1/bpartner/BpartnerRestControllerTest.snap

```diff
@@ -35,6 +35,7 @@ de.metas.rest_api.v1.bpartner.BpartnerRestControllerTest.createOrUpdateBPartner_
         "externalId": "123",
         "handoverLocation": false,
         "metasfreshId": 1000000,
+        "name": "bpartnerRecord.name_updated",
```

Comment:
changed because `de.metas.bpartner.interceptor.C_BPartner_Location` isn't registered in tests